tests/test_config_parsing/test_new_observations_parser.py

```diff
@@ -76,42 +76,9 @@ def test_parse():
     )
 
 
-def test_parse_include_outside_raises_obsconf_error():
+def test_that_unexpected_character_gives_observation_config_error():
     with pytest.raises(
         ObservationConfigError,
-        match="Observation parsing failed: Did not expect character",
+        match="Observation parsing failed: Did not expect character: i",
```

Comment:
Fixed, it now includes the line+phrase causing the error

Reply:
Very nice! Thanks! 
Could this be reflected in the test? e.g. instead of `match` the exact string, test that it also includes `include a` and `0` (the line number)?

Reply:
Fixed, (just a note on line/col numbers, as lark uses them with indexing starting at 1, we also follow this convention throughout the code, so the test requires it to be 1)